tests/unit/preprocessor/KernelPCA_unittest.cc

```diff
@@ -7,37 +7,66 @@
 using ::testing::Test;
 using namespace shogun;
 
-#ifdef HAVE_LAPACK
-TEST(KernelPCA, DISABLED_apply_to_feature_matrix_input)
+TEST(KernelPCA, apply_to_feature_matrix_input)
 {
-	float64_t data[] = {1, 1, 1,
-                      1, 2, 3,
-                      5, 6, 1,
-                      2, 2, 2,
-                      1, 1, 1};
-	float64_t resdata[] = {-1.526879008202007e-02,  6.902776989923266e-01,
-                         -4.032822763552926e-01, -5.151523890814317e-01,
-                         8.444041004961732e-01, -4.318711485273607e-01,
-                         -4.105842439768400e-01, -4.335318603758601e-01,
-                         -1.526879008202015e-02, 6.902776989923268e-01
-	                        };// column-wise
+	float64_t data[] = {1, 1, 1, 1, 2, 3, 5, 6, 1, 2, 2, 2, 1, 1, 1};
+	float64_t test_data[] = {3, 3, 3, 7, 4, 1};
+	// Results compared with sklearn
+	// https://gist.github.com/micmn/f93f723b74db2a1eb5875f63d841bdc1
+	float64_t resdata[] = {-0.17645841, 0.013962, -0.16082441, 0.03640145};
```

Comment:
I think we should from now on avoid any fix data creation code in unit tests, and always move it to function, fixture, or whatever. What do you think?